api/external/infra_proxy/infra_proxy.proto

```diff
@@ -35,150 +35,150 @@ service InfraProxy {
-    option (chef.automate.api.iam.policy).action = "infra:servers:get";
+    option (chef.automate.api.iam.policy).resource = "infra:chefServers:{name}";
+    option (chef.automate.api.iam.policy).action = "infra:chefServers:get";
   };
   rpc CreateServer (infra_proxy.request.CreateServer) returns (infra_proxy.response.CreateServer) {
     option (google.api.http).post = "/infra/servers";
     option (google.api.http).body = "*";
-    option (chef.automate.api.policy).resource = "infra:servers";
+    option (chef.automate.api.policy).resource = "infra:chefServers";
     option (chef.automate.api.policy).action = "create";
-    option (chef.automate.api.iam.policy).resource = "infra:servers";
-    option (chef.automate.api.iam.policy).action = "infra:servers:create";
+    option (chef.automate.api.iam.policy).resource = "infra:chefServers";
+    option (chef.automate.api.iam.policy).action = "infra:chefServers:create";
   };
   rpc UpdateServer (infra_proxy.request.UpdateServer) returns (infra_proxy.response.UpdateServer) {
     option (google.api.http).put = "/infra/servers/{id}";
     option (google.api.http).body = "*";
-    option (chef.automate.api.policy).resource = "infra:servers:{id}";
+    option (chef.automate.api.policy).resource = "infra:chefServers:{id}";
     option (chef.automate.api.policy).action = "update";
-    option (chef.automate.api.iam.policy).resource = "infra:servers:{id}";
-    option (chef.automate.api.iam.policy).action = "infra:servers:update";
+    option (chef.automate.api.iam.policy).resource = "infra:chefServers:{id}";
+    option (chef.automate.api.iam.policy).action = "infra:chefServers:update";
   };
   rpc DeleteServer (infra_proxy.request.DeleteServer) returns (infra_proxy.response.DeleteServer) {
     option (google.api.http).delete = "/infra/servers/{id}";
-    option (chef.automate.api.policy).resource = "infra:servers:{id}";
+    option (chef.automate.api.policy).resource = "infra:chefServers:{id}";
     option (chef.automate.api.policy).action = "delete";
-    option (chef.automate.api.iam.policy).resource = "infra:servers:{id}";
-    option (chef.automate.api.iam.policy).action = "infra:servers:delete";
+    option (chef.automate.api.iam.policy).resource = "infra:chefServers:{id}";
+    option (chef.automate.api.iam.policy).action = "infra:chefServers:delete";
   };
 
   rpc GetOrgs (infra_proxy.request.GetOrgs) returns (infra_proxy.response.GetOrgs) {
     option (google.api.http).get = "/infra/servers/{server_id}/orgs";
-    option (chef.automate.api.policy).resource = "infra:servers:{server_id}";
+    option (chef.automate.api.policy).resource = "infra:chefServers:{server_id}";
     option (chef.automate.api.policy).action = "read";
-    option (chef.automate.api.iam.policy).resource = "infra:servers:{server_id}:orgs";
-    option (chef.automate.api.iam.policy).action = "infra:serverOrgs:get";
+    option (chef.automate.api.iam.policy).resource = "infra:chefServers:{server_id}:orgs";
```

Comment:
since there's no other id in this action, it should be safe to use `id` here:
```suggestion
    option (chef.automate.api.iam.policy).resource = "infra:chefServers:{id}:orgs";
```